exo-go/src/service_helpers/index.go

```diff
@@ -27,3 +27,15 @@ func GetExistingServices(services types.Services) []string {
 	}
 	return existingServices
 }
+
+// GetServiceProtectionLevels returns a map containing service names to their protection level
+func GetServiceProtectionLevels(appConfig types.AppConfig) map[string]string {
```

Comment:
This file no longer exists, now we only have `app_config_helpers` and `service_config_helpers`.